### PR TITLE
Add link to Activity -> History Tab

### LINKED ISSUE #365: Items in Activity>History should link to their respective movie info pages
Small point but I'm running into it quite a bit right now.

### PR DESCRIPTION
**Pull request by fedoranimus**
Fixes #365

### CODE REVIEW

File: src/UI/Activity/Queue/QueueLayout.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 var Marionette = require('marionette');
 var Backgrid = require('backgrid');
 var QueueCollection = require('./QueueCollection');
-var SeriesTitleCell = require('../../Cells/MovieTitleCell');
+var MovieTitleCell = require('../../Cells/MovieTitleCell');
```
Comment (galli-leo) [inline src/UI/Activity/Queue/QueueLayout.js]: You are not using MovieTitleHistoryCell here. Might want to change that :wink:

Comment (fedoranimus) [inline src/UI/Activity/Queue/QueueLayout.js]: It works fine as written and allows itself to _properly_ use a route/title without hacks like the history cell does. No idea why, but it does.

Comment (galli-leo) [inline src/UI/Activity/Queue/QueueLayout.js]: Ah ok, in that case good to merge. (Just thought you ment to use that one as you changed that one.)

Comment (fedoranimus) [inline src/UI/Activity/Queue/QueueLayout.js]: Nah, was just changing the name.

**Pull request merged**